js/original.js

```diff
@@ -143,6 +145,70 @@
   function calculateScore() {
     score += Planets.length * Stars.length * 5;
     if (Planets.length == 0) score -= 1;
+    if (score > nextLevel && score < nextLevel + (nextLevel / 200)) {
+      levelUp();
+    }
+  }
+  function levelUp() {
+    let sunLocationX = randomInt(radius, innerWidth - radius);
+    let sunLocationY = randomInt(radius + 100, innerHeight - radius);
+    let existingX = [];
```

Comment:
Stars Array already exist there on [this line](https://github.com/nveenjain/gravity/pull/10/files#diff-51771aa99a43d67cef66bdda737d4fe0R22). You can check the stars and planets.